client-app/components/atoms/image-gallery/image-gallery.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    ref="mainImageDiv"
+    class="square relative flex flex-col justify-center items-center border border-gray-100 rounded-sm"
+  >
+    <img
+      :src="activeSrc"
+      alt="product.name"
+      class="absolute top-0 w-full h-full object-cover object-center rounded-sm"
+    />
+  </div>
+  <div v-if="isMobile && images && images.length > 1" class="mt-4 flex flex-row justify-center space-x-2.5">
+    <div
+      v-for="image in images"
+      :key="image?.url || ''"
+      class="border border-yellow-500 w-4 h-4 rounded-full cursor-pointer"
+      :class="{
+        'bg-yellow-500': image?.url == activeSrc,
+      }"
+      @click="setActiveImage(image?.url || '')"
+    ></div>
+  </div>
+  <div v-if="!isMobile && images && images.length > 1" class="mt-6 grid grid-cols-3 gap-5">
+    <div v-for="image in images" :key="image?.url || ''" @click="setActiveImage(image?.url || '')">
+      <div
+        class="square relative flex flex-col justify-center items-center cursor-pointer border border-gray-100 rounded-sm hover:ring hover:ring-yellow-200"
+        :class="{
+          'ring ring-yellow-500': image?.url == activeSrc,
+        }"
+      >
+        <img
+          :src="image?.url || '/static/images/no-image.png'"
```

Comment:
Image will be broken after merge. `no-image.svg` moved to `common/no-image.svg`. Please, review image sources and put new images into corresponding folders inside `assets/images`.